### PR TITLE
Update django admin

### PR DESCRIPTION
**Pull request by JackRyan1989**
Updated SEP_data, Behavioral_Health_Note and HCV_Note models in addition to changing the models listed in admin.py.

### CODE REVIEW

File: core/sep_data/views.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from core.viewsets import ModelViewSet
-from core.models import Sep_Data
+from core.models import SepData
 from core.sep_data.serializers import Sep_DataSerializer
 
 class Sep_DataViewSet(ModelViewSet):
```
Comment (MikeyManoguerra) [inline core/sep_data/views.py:5]: Update the view to match the model name

File: core/sep_data/serializers.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
 from rest_framework import serializers
-from core.models import Sep_Data
+from core.models import SepData
 
 
 class Sep_DataSerializer(serializers.ModelSerializer):
```
Comment (MikeyManoguerra) [inline core/sep_data/serializers.py:5]: Update the serializer to match the model name

**Review by MikeyManoguerra — CHANGES_REQUESTED** [review]
Looks good! Just a Small naming change.

While looking at the branch, I was reminded you can customize the link in the admin by defining the `__str__` method on the model class. You can see an example on the Visit model.

Lets get this merged in and we can open  a separate  issue for that.

https://docs.djangoproject.com/en/3.1/ref/models/instances/#django.db.models.Model.__str__

**Review by JackRyan1989 — COMMENTED** [review]
Updated admin.py, sep data model, behavioral health notes model, hcv notes model along with the sep data view and serializer.

**Review by JackRyan1989 — COMMENTED** [review]
Reviewed and addressed

**Review by MikeyManoguerra — APPROVED** [review]
Looks good, Thanks for the final changes

**Pull request merged**